### DISCUSSION TITLE
ToggleColumn Sizes

### DISCUSSION THREAD

**Discussion by sairiz** [filamentphp/filament#13243] [Help]
### Package

Table builder

### Package Version

v3.2.90

### How can we help you?

Hi, i have been trying to figure out if there is already a way to change the size of the ToggleColumn via a method or class.

So far i haven't found out any. The closest thing i found is the width() method but to no effect.

Any guidance would be much appreciated

**Comment by yuvrajtimalsina** [comment]
You may achieve it by implementing custom columns :

[https://filamentphp.com/docs/3.x/tables/columns/custom](url)

Also, since the toggle column is interactive, you may lookup ToggleColumn class and its views and extract the logic, design to your custom view and make modifications as per your need.

**Reply by sairiz** [reply]
I hope i wouldnt have to use custom columns but it seems i have to. Appreciate the guide though. Thanks
